File: src/main/java/sk/hudak/prco/manager/ErrorHandler.java
package sk.hudak.prco.manager;

import sk.hudak.prco.dto.product.ProductDetailInfo;
import sk.hudak.prco.manager.impl.UpdateProcessResult;

public interface ErrorHandler {
    //FIXME premenovat
    UpdateProcessResult processParsingError(Exception error, ProductDetailInfo productDetailInfo);
}

File: src/main/java/sk/hudak/prco/manager/ErrorHandler.kt
package sk.hudak.prco.manager

import sk.hudak.prco.dto.product.ProductDetailInfo
import sk.hudak.prco.manager.impl.UpdateProcessResult

interface ErrorHandler {
    //FIXME premenovat
    fun processParsingError(error: Exception, productDetailInfo: ProductDetailInfo): UpdateProcessResult
}
